hw2/03-count.html

```diff
@@ -10,21 +10,21 @@
     />
     <script defer src="03-count.js"></script>
   </head>
-  <body style="background-color: #21495F" >
+  <body style="background-color: #21495f">
     <main>
       <div class="container my-5 text-center">
         <div
           class="card bg-light mb-5 mx-auto"
           style="max-width: 48rem; height: 26rem"
         >
-          <h1 class = "text-center my-3"><strong>Exercise 03 - Count</strong></h1>
+          <h1 class="text-center my-3"><strong>Exercise 03 - Count</strong></h1>
           <hr />
           <div>
-            <label >Highlight all occurences of a word:</label>
-            <input id = "search"/>
+            <label>Highlight all occurences of a word:</label>
```

Comment:
Adding for attribute to the label will help you to resolve the accessibility error.

Reply:
thanks for pointing out the accessibility